helm/values.yaml

```diff
@@ -236,7 +236,7 @@ worker:
       # You can disable the default fleet options
       # and set your own.
       enabled: true
-      plugins: "javascript,python"
+      plugins: "javascript"
```

Comment:
Let’s change the fleet name? OPA maintainers are probably okay with a momentary blip as the new deployment is rolled out

Reply:
They would need to also update their values.yaml if they updated any scaling options.